specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json

```diff
@@ -522,16 +522,22 @@
           "description": "The status of the email",
```

Comment:
Update to mention that any value other than "Delivered" is a failure to deliver.